ovirt/resource_ovirt_vm.go

```diff
@@ -517,6 +518,47 @@ func resourceOvirtVMUpdate(d *schema.ResourceData, meta interface{}) error {
                 return err_updateresult
         }
 
+	// Check status and Start/Stop VM
+	status, statusOK := d.GetOk("status")
+
+	if d.HasChange("status") && statusOK {
+	        // Try to start VM
+	        log.Printf("[DEBUG] Try to update runing status for VM (%s)", d.Id())
+		var vm_status ovirtsdk4.VmStatus
+
+		if status == "up" {
```

Comment:
I rather have a switch case here, to catch more statuses in the future (pause?)

Reply:
Changed code to the switch:
```                switch status {
                case "up":
                        vm_status = ovirtsdk4.VMSTATUS_UP
                        _, err = vmService.Start().Send()
                case "down":
                        vm_status = ovirtsdk4.VMSTATUS_DOWN
                        _, err = vmService.Stop().Send()
                }
                if err != nil {
                        log.Printf("[DEBUG] Failed to change status for VM (%s)", d.Id())
                        return err
                }
```

Commit - https://github.com/oVirt/terraform-provider-ovirt/pull/198/commits/7fa45520ca4c370cc765d9bd9a0883fea7e409ce